src/textbracket.ts

```diff
@@ -119,7 +108,13 @@ export class TextBracket extends Element {
    * @returns this
    */
   applyStyle(ctx: RenderContext): this {
-    ctx.setFont(this.font);
+    this.textElement.setFont(this.textFont);
+    // We called this.resetFont() in the constructor, so we know this.textFont is available.
+    const { family, size, weight, style } = this.textFont;
+    // To draw the superscript, we scale the font size by 1/1.4.
```

Comment:
I see, it is 1/1.4. Should this be customizable?

Reply:
I suggest to make a general review on constants but not as part of this PR. The super and sub indexes should have, I believe, a common ratio (textbrackets, chordsymbols,...)